.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```

Comment:
Don't include these files. Please delete them and then push the changes.

Reply:
Also remove the __pycache__ files... As a general rule, never commit something that you personally did not edit.